### ISSUE TITLE
Asset compilation issue

### ISSUE THREAD

**Issue by Shidash** [TransparencyToolkit/LookingGlass#43]
When running rake assets:precompile in production, I get this error-
Sass::SyntaxError: Undefined variable: "$btn-border-radius-base".
  (in /var/www/update/LookingGlass/app/assets/stylesheets/bootstrap-custom.scss:20)
/var/www/update/LookingGlass/vendor/bundle/ruby/2.2.0/gems/bootstrap-sass-3.3.5/assets/stylesheets/bootstrap/_buttons.scss:20:in `button-size'
/var/www/update/LookingGlass/vendor/bundle/ruby/2.2.0/gems/bootstrap-sass-3.3.5/assets/stylesheets/bootstrap/_buttons.scss:20
/var/www/update/LookingGlass/app/assets/stylesheets/bootstrap-custom.scss:17
I recall that we had this issue before. How can this be fixed? 

Also, the logo in the upper left isn't showing up- what file/icon should be in the instance spec for that?

Thanks

**Comment by bnvk** [comment]
I just pushed a fix that adds this in 0df5e6b4fe467e41c4dc9c555b9fb6335d863220

> Also, the logo in the upper left isn't showing up- what file/icon should be in the instance spec for that?

Make sure the items in the `instances/config.json` file are the following:

```
    "Favicon": "tt-favicon.png",
    "Logo": "tt-logo.png",
    "Theme": "theme-default",
```

See the updated [example-config.json](https://github.com/TransparencyToolkit/LookingGlass/blob/master/app/dataspec/instances/example-config.json#L5-L7) file with these values

**Comment by bnvk** [comment]
@Shidash does the above resolve the issue? I ran into Bethany and she said you *just* 45 mins ago found another issue!
